Makefile

```diff
@@ -12,8 +12,8 @@ all: libs
 	./scripts/go_executable_build.sh
 
 libs:
-	make -C $(TOP)/mcl -j4
```

Comment:
I removed -j4 because we should not assume that the machine running the build has 4 core cpus.
t3.small which is our min requirement has only 2.